check.go

```diff
@@ -198,9 +202,6 @@ func (c *C) logNewLine() {
 
 func (c *C) writeLog(buf []byte) {
```

Comment:
When the verbosity is set to 2 this method would write to both the log and the stream, right? If that's the case maybe we could put a comment, or call a downstream method with a clear indication of what's about to happen, this name can be misleading.